src/common/backend/utils/cache/catcache.cpp

```diff
@@ -3296,3 +3296,29 @@ SetCatCacheList(CatCache *cache,
 
     DLAddHead(&cache->cc_lists, &cl->cache_elem);
 }
+
+/*
+ *	RelationHasCachedLists
+ *
+ *	Returns true if there is a catalog cache associated with this
+ * 	relation which is currently caching at least one list.
+ */
+bool RelationHasCachedLists(Relation relation)
+{
+    CatCache* ccp = NULL;
+	Oid reloid;
+
+	/* sanity checks */
+	Assert(RelationIsValid(relation));
+    Assert(u_sess->cache_cxt.cache_header != NULL);
```

Comment:
there seems to be some indentation issue here. maybe tabs vs spaces?

Reply:
yes, seems caused by tabs vs spaces. fixed. thanks